src/main/java/tech/jhipster/lite/module/infrastructure/secondary/FileSystemPackageJsonHandler.java

```diff
@@ -74,6 +81,10 @@ private Path getPackageJsonFile(JHipsterProjectFolder projectFolder) {
     return file;
   }
 
+  private String replacePlaceholders(JHipsterModuleContext context, String content) {
```

Comment:
If find it more logical that way, but this is probably subjective :)
```suggestion
  private String replacePlaceholders(String content, JHipsterModuleContext context) {
```

BTW: this is the signature you used in `FileSystemReplacer`

Reply:
I agree. I copied the method from ...PackageJson...handler and felt strange. It is better to make the method signature logic inside the class context since it is a private method.

Reply:
> I agree. I copied the method from ...PackageJson...handler and felt strange. It is better to make the method signature logic inside the class context since it is a private method.

Just to share my logic was to follow the other methods that put content in the last parameter:

```java
    String content = readContent(file);
    content = replaceType(indentation, packageJson.type(), content);
    content = replaceScripts(indentation, packageJson.scripts(), content);
    content = replaceDevDependencies(indentation, packageJson.devDependencies(), content);
    content = replaceDependencies(indentation, packageJson.dependencies(), content);
    content = removeDependencies(indentation, packageJson.dependenciesToRemove(), content);
    content = removeDevDependencies(indentation, packageJson.devDependenciesToRemove(), content);

    content = replacePlaceholders(packageJson.context(), content);
    content = cleanupLineBreaks(indentation, content);
    ```

I will change it, no problem 👍